v2/datastream-to-spanner/terraform/samples/monitoring-alerting/main/README.md

```diff
@@ -0,0 +1,63 @@
+# Spanner Migration Tool Monitoring Dashboard - Terraform Module
```

Comment:
Where possible, can you mirror the `sample` structure we have for other templates? 

Ref - https://github.com/GoogleCloudPlatform/DataflowTemplates/tree/main/v2/datastream-to-spanner/terraform/samples/mysql-sharded-end-to-end

Example: We include a `terraform.tfvars` and `terraform_simple.tfvars` for each sample. Definitions are here - https://github.com/GoogleCloudPlatform/DataflowTemplates/tree/main/v2/datastream-to-spanner/terraform/samples#sample-structure


As much as possible, please try to keep the naming convention consistent.

Reply:
Instead of using "Spanner Migration Tool" as a way to refer, could you please replace it with "Live migration template(s)".

For example - 

1. Live Migration Monitoring Dashboard (Title)
2.  `visualize key metrics related to the Live migration template(s)` (first line)

Reply:
Updated the `README.md` and the dashboard title to use `Live Migration Monitoring Dashboard`